src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header-content {
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+li {
+  display: inline-block;
+  margin: 0 15px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.logo-link {
+  display: block;
+  height: 40px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: rgba(0, 0, 0, 1);
+}
+
+.nav__link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
```

Comment:
add styles to `is-active`. 
`position: absolute` and `color`

Reply:
you can simplify the selector by removing nav__link class from it.

Reply:
Updated. Thanks for the suggestion.
10382f7918d9effcd44b7a5de593421554c60af2